hack/helm/values.yaml

```diff
@@ -0,0 +1,16 @@
+nameOverride: ""
+fullnameOverride: ""
```

Comment:
Comment these, i.e. what are the default values ("hnc"?) and what do they control?

Reply:
added a comment. This is the current default that get's provided when one does a `helm create` which I'll explain in more detail here and you can tell me if the comment is suitable.

`helm install 123 .` sensible defaults apply and objects get named `hnc-123` which is a combination of Chart.yaml name and the release name
`helm install --set nameOverride=foo 123` objects get named `foo-123`
`helm install --set fullnameOverride=bar 123` objects get named `bar`